frontend/src/components/widgets/DateInput/DateInput.tsx

```diff
@@ -106,7 +108,17 @@ class DateInput extends React.PureComponent<Props, State> {
 
     return (
       <div className="stDateInput" style={style}>
-        <StyledWidgetLabel>{element.label}</StyledWidgetLabel>
+        <StyledWidgetLabel>
```

Comment:
Hm, this applies both to this PR and the other similar ones, but should we consider adding a helper function (or a small wrapper component) that generalizes this code more?

The code below seems replicated a good amount and could be made a single line in most places with a bit of refactoring (although it seems like there are a few places where things slightly differ).